docs/configuration.md

```diff
@@ -78,6 +78,98 @@ TOML is also supported in `.release-it.toml`:
+| `npm.ignoreVersion` | When set to `true`, ignore the `version` from `package.json`. |
+| `npm.allowSameVersion` | If set to `true`, prevents throwing error when setting the new version to the same value as the current version. Note it is recommended to use `versionArgs` for this instead. |
+| `npm.versionArgs` | In case extra arguments should be provided to npm for the versioning operation. |
+| `npm.skipChecks` | If set to `true`, skip checks on whether the npm registry is up, the user is authenticated with npm and is a collaborator for the current package. |
+| `npm.timeout` | Timeout duration to wait for a response from the npm registry. |
+
+### GitHub
+
+| Option | Description |
+| --- | --- |
+| `github.release` | Set to `false` to skip the GitHub publish step. |
+| `github.releaseName` | Set the release name, which is `Release ${version}` by default. |
+| `github.releaseNotes` | Override the release notes with custom notes. |
+| `github.autoGenerate` | Set to `true` to let GitHub generate release notes. |
+| `github.preRelease` | Set to `true` to set the release to a pre-release status. |
+| `github.draft` | Set to `true` to set the release to a draft status. |
+| `github.tokenRef` | Set this to a string to change the name of the GitHub token environment variable (defaults to `GITHUB_TOKEN`). |
+| `github.assets` | Glob pattern path to assets to add to the GitHub release. |
+| `github.host` | Use a different host from what would be derived from the Git URL (e.g. when using GitHub Enterprise). |
+| `github.timeout` | Timeout duration to wait for a response from the GitHub API. |
+| `github.proxy` | If the release is performed behind a proxy, set this to string of the proxy URL. |
+| `github.skipChecks` | If set to `true`, skip checks on whether the `GITHUB_TOKEN` environment variable is set, will authenticate, and verify whether the current user is a collaborator and authorized to publish a release. |
+| `github.web` | Set to `true` to explicitly override checking if the `GITHUB_TOKEN` is set. |
+| `github.comments.submit` | Set to `true` to submit a comment to each merged pull requests and closed issues that are part of the release. |
+| `github.comments.issue` | The text to add to the associated closed issues. |
+| `github.comments.pr` | The text to add to the associated merged pull requests. |
+
+### GitLab
+
+| Option | Description |
+| --- | --- |
+| `gitlab.release` | Set to `false` to skip the GitLab publish step. |
+| `gitlab.releaseName` | Set the release name, which is `Release ${version}` by default. |
+| `gitlab.releaseNotes` |  Override the release notes with custom notes. |
+| `gitlab.milestones` | To associate one or more milestones with a GitLab release, set the `gitlab.milestones` option to an array of the titles of the corresponding milestones. |
+| `gitlab.tokenRef` |  Set this to a string to change the name of the GitHub token environment variable (defaults to `GITLAB_TOKEN`). |
```

Comment:
"GitHub"

Reply:
Fixed in https://github.com/release-it/release-it/pull/1155/commits/e3c93ed9bb2a75bebb93be57eaad237d9fa9f23a